taxcalc/calculate.py

```diff
@@ -26,18 +27,24 @@ def add_df(alldfs, df):
 class Calculator(object):
 
     def __init__(self, policy=None, records=None,
-                 sync_years=True, behavior=None, **kwargs):
+                 sync_years=True, behavior=None, growth=None, **kwargs):
 
         if isinstance(policy, Policy):
             self._policy = policy
         else:
             msg = 'Must supply tax parameters as a Policy object'
             raise ValueError(msg)
+
         if isinstance(behavior, Behavior):
             self.behavior = behavior
         else:
             self.behavior = Behavior(start_year=policy.start_year)
 
+        if isinstance(growth, Growth):
+            self.growth = growth
+        else:
+            self.growth = Growth(start_year=policy.start_year)
```

Comment:
Hmm.. this idiom seems bad to me (used for both Growth and Behavior). If someone passes a Growth object for `growth`, then we set it to `self.growth`. If they pass anything else (a string, some other kind of object, whatever), we just silently make a new Growth object and assign it to `self.growth`. Basically the user could assume he/she is sending in an object for `growth` but actually ending up with something else (with no error/warning message). That seems bad. How about something like this?
```
if growth:
    if isinstance(growth, Growth):
        self.growth = growth
    else:
        raise ValueError("Must supply growth as a Growth object")
else:
    self.growth = Growth(start_year=policy.start_year
```